### ISSUE TITLE
Button are not working in tabview 

### ISSUE THREAD

**Issue by jitenderchand1** [react-native-community/react-native-tab-view#604]
### Current behaviour

Button inside a tab views not working. 

### Expected behaviour

The button should work inside tab view

### Code sample

```import * as React from 'react';
import { View, StyleSheet, Dimensions, Button } from 'react-native';
import { TabView, TabBar, SceneMap } from 'react-native-tab-view';

const FirstRoute = () => (
  <View style={[{ backgroundColor: '#ff4081' }]}><Button onPress={()=>console.warn('test')} title="click" /></View>
);
const SecondRoute = () => (
  <View style={[{ backgroundColor: '#ff4081' }]}><Button onPress={()=>console.warn('test')} title="click" /></View>
);

export class TabbedView extends React.Component {
  state = {
    index: 0,
    routes: [
      { key: 'first', title: 'First' },
      { key: 'second', title: 'Second' },
    ],
  };

  render() {
    return (
      <TabView
        navigationState={this.state}
        renderScene={SceneMap({
          first: props=> <FirstRoute {...props} />,
          second: props=> <SecondRoute {...props}/>,
        })}
        onIndexChange={index => this.setState({ index })}
        initialLayout={{ width: Dimensions.get('window').width }}
      />
    );
  }
}

```
### Screenshots (if applicable)

NA

### What have you tried
The issue is happening only when I am passing props in tab view. for eg.
```
first: props=> <FirstRoute {...props} />,
```
it has no issue when I am using routes without props for eg:

```
first: FirstRoute,
```

### Your Environment

| software              | version
| --------------------- | -------
| ios         | macOS Sierra 10.12
| react-native  | sdk-27.0.0.tar.gz
| react-native-tab-view | ^1.0.0
| node  | 8.9.1
| npm   |  5.5.1

**Comment by jitenderchand1** [comment]
Can you please look into this issue

**Comment by jitenderchand1** [comment]
@ManigandanRaamanathan No, I created my own tabs with very basic feature.

**Comment by missRuoHua** [comment]
+1

**Comment by huaeval** [comment]
> +1

TouchableOpacity>TouchableOpacity.delete one TouchableOpacity.it's working!Good luck to you.

**Comment by jitenderchand1** [comment]
@huaeval I am sorry but I didn't understand what do you mean. Can you please elaborate your answer.

**Comment by arirah** [comment]
Write a function like 

_renderScene = ({ route }) => {
    switch (route.key) {
      case 'first':
        return <Component1 navigation={this.props.navigation} />;
      case 'second':
        return <Component2  navigation={this.props.navigation} />;
      default:
        null;
    }
  };


Then call it to 
 <TabView
              navigationState={this.state}
              renderScene={ this._renderScene} 
...................
..........

I tested it with button on press to navigate another screen , it works !!

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
